### PR TITLE
Web Almanac 2020: Add web story for first chapter

### PR DESCRIPTION
**Pull request by swissspidy**
I used a small scraper script to download the story created in WordPress and adapt it for this site's structure. The markup is unfortunately not easy to read because it's all auto-generated from that story editor tool.

Not sure if it works though, as I couldn't test it. When I got the local site running and tried to access `http://127.0.0.1:8080/en/2020/stories/web-almanac-2020-page-content`, I get an `404 Not Found: Unsupported chapter requested` error.

See #1571

### CODE REVIEW

**Comment by bazzadp** [comment]
Just submitted some fixed to make this work:  https://github.com/HTTPArchive/almanac.httparchive.org/pull/1800/commits/41366824472746d87e81589d9815cfc758a96a2e

Should now be able to run it locally here: http://127.0.0.1:8080/en/2020/stories/web-almanac-2020-page-content

Also available here: https://20201218t151308-dot-webalmanac.uk.r.appspot.com/en/2020/stories/web-almanac-2020-page-content

**Comment by swissspidy** [comment]
Awesome, thank you so much! 💪 🎉 🚀

**Comment by bazzadp** [comment]
Add some more fixes and restaged it here: https://20201218t153239-dot-webalmanac.uk.r.appspot.com/en/2020/stories/web-almanac-2020-page-content

Need to dash off not but pop any questions in here and can look over the weekend.

File: src/templates/en/2020/stories/web-almanac-2020-page-content/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
..."${title}","send_to":"UA-22381566-3"}},"audioMuted":{"on":"story-audio-muted","request":"event","vars":{"event_name":"custom","event_action":"story_audio_muted","event_category":"${title}","send_to":"UA-22381566-3"}},"audioUnmuted":{"on":"story-audio-unmuted","request":"event","vars":{"event_name":"custom","event_action":"story_audio_unmuted","event_category":"${title}","send_to":"UA-22381566-3"}},"pageAttachmentEnter":{"on":"story-page-attachment-enter","request":"event","vars":{"event_name":"custom","event_action":"story_page_attachment_enter","event_category":"${title}","send_to":"UA-22381566-3"}},"pageAttachmentExit":{"on":"story-page-attachment-exit","request":"event","vars":{"event_name":"custom","event_action":"story_page_attachment_exit","event_category":"${title}","send_to":"UA-22381566-3"}},"storyBookendEnter":{"on":"story-bookend-enter","request":"event","vars":{"event_name":"custom","event_action":"story_bookend_enter","event_category":"${title}","send_to":"UA-22381566-3"}},"storyBookendExit":{"on":"story-bookend-exit","request":"event","vars":{"event_name":"custom","event_action":"story_bookend_exit","event_category":"${title}","send_to":"UA-22381566-3"}}}}			</script>
```
Comment (rviscomi) [inline src/templates/en/2020/stories/web-almanac-2020-page-content/index.html:32]: Whose GA is this?

**Review by rviscomi — COMMENTED** [review]
Thanks for adding this @swissspidy!

Besides adding this to our sitemap, is there anything else needed to get this SEO-friendly?

File: src/templates/en/2020/stories/web-almanac-2020-page-content/index.html (hunk 1 above)
Comment (swissspidy) [reply src/templates/en/2020/stories/web-almanac-2020-page-content/index.html:32]: I thought I added the one from the Web Almanac site? Let me double check if not

**Comment by swissspidy** [comment]
> Thanks for adding this @swissspidy!
> 
> Besides adding this to our sitemap, is there anything else needed to get this SEO-friendly?

Don‘t think so!

File: src/templates/en/2020/stories/web-almanac-2020-page-content/index.html (hunk 1 above)
Comment (rviscomi) [reply src/templates/en/2020/stories/web-almanac-2020-page-content/index.html:32]: Oh ok I didn't know you had grabbed it from our site, all good then. I thought maybe it was a default from the editor. I hadn't checked to see if it was our ID first.

**Review by rviscomi — APPROVED** [review]

**Comment by swissspidy** [comment]
Thanks a lot for your help here @bazzadp @rviscomi 

I see you've created a few follow-up PRs to this, which I'll keep in mind for next time.

One suggestion that I have:

Remove the `X-Frame-Options` response header for stories, as otherwise stories can't be embedded in an `<amp-story-player>` and similar.

**Comment by bazzadp** [comment]
OK will do. Will also need to set `frame-ancestors https://*;` in [our stories specific CSP](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/server/stories_csp.py) as that basically is the new X-Frame-Options.

**Comment by bazzadp** [comment]
@swissspidy was thinking about this more. Can you explain why this is needed? Wouldn’t any embeds come from the AMP CDN url which I understand removes this?

Not a problem to add but more curious as to why it’s needed and can’t find any documentation on this.

**Comment by swissspidy** [comment]
@bazzadp Anyone can use `<amp-story-player>`, which is essentially an `<iframe>` with some story-specific controls (play, mute/unmute), on their website to embed a given story via its URL. So I'd embed https://almanac.httparchive.org/en/2020/stories/page-content directly, and not go via AMP DN or anything.

**Pull request merged**